README.md

```diff
@@ -751,11 +751,11 @@ Developers in the project may not follow consistent code security practices, lea
 
 ## ![✔] 6.10. Validate the incoming JSON schemas
 
-**TL;DR:** Whether user input is used in querying the database, making API calls, or accessing system files - all user input should always be validated to expected type. You can use [validator.js](https://github.com/chriso/validator.js/) or [joi](https://github.com/hapijs/joi) to validate general user input or json schemas. The top frontend libraries handle output encoding well, but nevertheless you should always make sure to output encode your user generated data in the correct context, and can make use of libraries such as [node-esapi](https://github.com/ESAPI/node-esapi) or [escape-html](https://github.com/component/escape-html).
+**TL;DR:** Validate the incoming requests` body payload and ensure it qualifies the expectations, fail fast if it doesn't. To avoid tedious validation coding within each route you may use lightweight JSON-based validation schemas such as [jsonschema](https://www.npmjs.com/package/jsonschema) or [JOI](https://www.npmjs.com/package/joi)
 
-**Otherwise:** Failure to encode user generated when outputting it can result in XSS, Log Injection or other vulnerabilities. Input validation should always be performed to confirm one is working with expected types and data properties (length, range, etc).
+**Otherwise:** Your generosity and permissive approach greatly increases the attack surface and encourage the attacker to tryout many inputs until it finds some combination that crashes the application
```

Comment:
I think it's worth mentioning that not validating JSON schema can lead to different things vulnerabilities like XSS, Parameter Pollutions, as well as indeed result in an application crash when not handled correctly.

Reply:
@lirantal will add that (application crash already mentioned)

p.s. I split this into two bullets (wrote the second only thus far):
6.9 *Generic sanitizing and escaping
6.10 *Per request validation (json schema, etc)

Reply:
👍